### ISSUE TITLE
Audio delay over 1000ms

### ISSUE THREAD

**Issue by kabud1321** [rom1v/sndcpy#103]
I recently tried scrcpy and very satisfied but when i tried sndcpy i got audio delay over 1000ms, I've tried a few things but didn't solve the problem. 
My devices Redmi note 8, i've tried a few different phone but facing same problem

**Comment by rom1v** [comment]
Over USB?

**Comment by kabud1321** [comment]
> Over USB?

Yes, im using scrcpy and sndcpy over usb, scrcpy working good and very low latency, only sndcpy i got high latency

**Comment by Chrombastic** [comment]
Have the same problem here, using a Samsung A50 (A505-F).

**Comment by xofficialgamer** [comment]
So did you found something that works with it?

**Comment by Zero3K** [comment]
Try increasing the network caching option in its .bat from 50 to 100.

**Comment by Shigeo87** [comment]
put network-caching to -1 on both sndcpy file and sndcpy.bat . it works for me.
![Screenshot 2021-09-12 203825](https://user-images.githubusercontent.com/90558900/132988078-2b8b7f0d-87b7-46b4-bcf7-8d3f16310a7d.png)
![Screenshot 2021-09-12 203756](https://user-images.githubusercontent.com/90558900/132988080-f4d921d4-d5c7-45b0-b2f1-40eb9fcbab48.png)

**Comment by ameaninglessname** [comment]
I encounter this issue while connecting wireless, and it is intended.
see:
https://github.com/rom1v/sndcpy/issues/150#issuecomment-890358852

**Comment by ameaninglessname** [comment]
I increase the display buffer of scrcpy with "--display-buffer=800", and now they are in sync.
Going to try something you're talking about.

**Comment by ameaninglessname** [comment]
put network-caching to -1 on sndcpy.bat . it works for me.

**Comment by MauNi-M** [comment]
not sure how to suggest changes to the code but I found that adding --clock-synchro=1 to the sndcpy.bat and sndcpy fixes the issue

**Comment by skygate2012** [comment]
@MauNi-M Thanks! This worked well!
